setropts/advanced/options_operators.md

```diff
@@ -38,15 +38,15 @@ Options use the following syntax: `<segment>:<options>`
 &nbsp;
 
 | **Option** | **Description** | **Valid Types** |
-| `base:raclist` | Add a class to the list of **raclisted** classes in RACF.  | `str` |
-| `base:audit_classes` | Add a class to the list of **audited** classes in RACF. | `str` |
-| `base:active_classes` | Add a class to the list of **active** classes in RACF.  | `str` |
-| `base:statiscs_class` | Track **statistics** for a specicified class.  | `str` |
-| `base:general_command_classes` | Allow **generic profile command** processing for a specified class. | `str` |
-| `base:generic_profile_checking_classes` | Allow **generic profile checking** for a specified class. | `str` |
-| `base:generic_profile_sharing_classes` | Allow sharing of in-storage **generic profiles** for a specified class. | `str` |
-| `base:global_access_classes` | Add a class to the list of **global access checking** classes in RACF.  | `str` |
-| `base:refresh` | **Refresh** the in-storage profiles for Genric, Global, or Raclist classes. | `bool` |
+| `base:raclist` | Add class(es) to the list of **raclisted** classes in RACF.  | `str` |
+| `base:audit_classes` | Add class(es) to the list of **audited** classes in RACF. | `str` |
+| `base:active_classes` | Add class(es) to the list of **active** classes in RACF.  | `str` |
+| `base:statiscs_class` | Track **statistics** for specicified class(es).  | `str` |
```

Comment:
`base:statistics_classes`

Reply:
Good catch, sorry that one was missed. changed.